feature-libs/organization/unit-order/components/unit-level-order-detail/unit-level-order-overview/unit-level-order-overview.component.html

```diff
@@ -21,29 +21,25 @@
       ></cx-card>
     </div>
 
-    <ng-container
-      *ngIf="order.purchaseOrderNumber || order.purchaseOrderNumber === ''"
-    >
-      <div class="cx-summary-card">
-        <cx-card
-          [content]="getUnitNameCardContent(order.orgUnit?.name) | async"
-        ></cx-card>
+    <div class="cx-summary-card">
```

Comment:
This needs to be feature flagged, I believe? You need to keep the conditional but also check for the version.